test/featureTestRunner.js

```diff
@@ -121,19 +130,27 @@ function setOptions(load, prologOptions) {
   load.metadata.traceurOptions = traceurOptions;
 }
 
-function featureTest(name, url) {
+function featureTest(name, url, overrideOptions = undefined) {
   test(name, (done) => {
     let baseURL = './';
 
     let prologOptions;
     function translateSynchronous(load) {
       let source = load.source;
+
       // Only top level file can set prologOptions.
```

Comment:
I think this comment is unclear or at least incomplete. Here would be a good spot to explain the two pass testing.